src/NetworkClient.ts

```diff
@@ -118,12 +135,9 @@ export default class NetworkClient {
   /* eslint-disable no-var */
   async post<R>(relativePath: string, data: any, query: Record<string, any> = {}): Promise<R | true> {
     try {
-      var response: AxiosResponse = await this.axiosInstance.post(`${relativePath}${stringifyQuery(query)}`, data);
+      var response = await this.axiosInstance.post(`${relativePath}${stringifyQuery(query)}`, data);
```

Comment:
As discussed offline, might want to replace all these try/catch blocks with a `.catch(throwApiError)`.